frontend/src/breeding_form.tsx

```diff
@@ -57,21 +64,30 @@ const emptyBreeding: Breeding = {
 export function BreedingForm({
   data,
   herdId,
+  handleBreedingsChanged,
+  handleActive,
 }: {
   data: Breeding | "new";
   herdId: string | undefined;
+  handleBreedingsChanged: any;
+  handleActive: any;
```

Comment:
Why any? They sound like a boolean.

Reply:
Nope, they are functions. But not "any" anyway, so I changed it. Thanks!